### PR TITLE
[DEVOPS-338] More complete travis sh

### PR DESCRIPTION
**Pull request by deepfire**
https://issues.serokell.io/issue/DEVOPS-338

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,6 +21,8 @@ smart-gen-*
 static/id_buildfarm
 *prof
 keys/*.key
+/test-*.yaml
```
Comment (domenkozar) [inline .gitignore]: What's this about?

Comment (domenkozar) [reply .gitignore]: Aren't those always deleted by `cleanup` function?

Comment (deepfire) [reply .gitignore]: They are, but it still causes a turmoil in your git status while the test is running.. : -)

Comment (domenkozar) [reply .gitignore]: Ideally then we'd use `mktemp -d` for intermediate artifacts and cleanup the directory after everything is done

Comment (deepfire) [reply .gitignore]: I agree, this could be the next improvement we make.

**Comment by domenkozar** [comment]
@deepfire CI fails

**Comment by domenkozar** [comment]
@deepfire this hasn't passed CI yet, why did you merge it?

**Comment by deepfire** [comment]
@domenkozar, it was an unfortunate accident -- I've had to have this locally, because otherwise `static/github_token` was in the `git status`.

**Comment by deepfire** [comment]
@domenkozar, but I think there is no reason for worry:
1. if it doesn't work I'm fixing it as I go
2. Travis doesn't build much of anything today, anyway : -/

**Comment by domenkozar** [comment]
How is that a blocker or a real problem? :)

**Comment by domenkozar** [comment]
I'd like for workflows to be respected, things must pass CI

**Comment by deepfire** [comment]
@domenkozar, I totally agree -- it was an unfortunate accident, as I said.

**Comment by domenkozar** [comment]
If things are accidentally merged, they should be reverted.

**Comment by deepfire** [comment]
@domenkozar, if you think it would be efficient use of our time to do this -- of course, I can do that.

**Comment by domenkozar** [comment]
Yes, I think respecting the workflows is an efficient use of time.

**Pull request merged**